pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -123,11 +109,21 @@ func (n *NodeLabeller) execute() bool {
 	defer n.queue.Done(key)
 
 	if !n.kvmController.NodeHasDevice(kvmPath) {
-		n.logger.Errorf("node-labeller cannot work without KVM device.")
+		if !n.kvmPostedError {
```

Comment:
Is this here to prevent multiple prints because the labelling mechanism runs indefinitely?

Reply:
yes exactly